zowe/zowe-cli
# Enhancing GetJobs.getJob

**What It Does**
Adds "exec-started" and "exec-ended" to IJob return data from GetJobs.getJob

**How to Test**
Download branch and test a command that uses GetJobs.getJob in the background - set a breakpoint and ensure that the job being returned has the expected timestamps. 

**Review Checklist**
I certify that I have:


**Additional Comments**
Other exec properties were included in the zosmf response from GetJobs. Should we include these other properties starred in the image? @t1m0thyj @zFernand0 @gejohnston @awharn 
![Screenshot 2024-10-23 at 2 20 51 PM](https://github.com/user-attachments/assets/3030a491-1ad8-4c3f-9333-cf09a56b6b94)
